core/server/api/canary/members.js

```diff
@@ -26,8 +26,50 @@ const cleanupUndefined = (obj) => {
     }
 };
 
-// NOTE: this method can be removed once unique constraints are introduced ref.: https://github.com/TryGhost/Ghost/blob/e277c6b/core/server/data/schema/schema.js#L339
-const sanitizeInput = (members) => {
+const sanitizeInput = async (members) => {
```

Comment:
This whole method could probably go live as a separate module in members importer. The parameter which it would need to get though is all the stuff needed for JSON schema validation.